client/src/components/content/HeaderNavbar/HeaderNavbar.jsx

```diff
@@ -18,6 +18,11 @@ const HeaderNavbar = (props) => {
     onOverview = false;
   }
 
+  let bpmnModelerLink = '/modeler';
+  if (selectedKey === 2 || selectedKey === 3) {
```

Comment:
I've already commented this as a question before, but could we maybe give these magic numbers some meaning? maybe save them to constants?

Reply:
Good point. I've made a constant for each Key (Number). @SanJSp